### ISSUE TITLE
Dynamic source rendering for Angular

### ISSUE THREAD

**Issue by sashafklein** [storybookjs/storybook#10617]
I'm using the Docs addon in an Angular project. 

It's generally working great, but when I click to see the code which generates the component, I instead am shown the code which generates the **story**:

As in, for this code in my stories file, 

```ts
export const Light = () => ({
  component: ButtonsComponent,
  props: {
    class: 'button--light',
    linkUrl: 'https://www.google.com',
  },
});

Light.story = {
  parameters: {
    docs: {
      iframeHeight: 80,
    },
  },
};
```

I see this when I view the storybook docs page and click "Show Code":

![image](https://user-images.githubusercontent.com/1746778/80845981-c21e4700-8bbf-11ea-8dfc-9b896d85a381.png)

Because the story docs should ideally be helpful for easily copying components into regular apps, I wanted to clean up this code and make it more similar to how components would be used in apps. 

First, I did a refactor, creating a `makeStory` function, which abstracted away some of the repeated code with sensible defaults, and allowed the declaration to be:

```ts
const Button = makeStory(ButtonsComponent);
export const Light = Button({ class: 'button--light', linkUrl: 'https://www.google.com' });

// And
export const Dark = Button({ class: 'button--dark', linkUrl: 'https://www.google.com' })
```

That's better:

![image](https://user-images.githubusercontent.com/1746778/80846109-24774780-8bc0-11ea-971e-d5ce8550c3f0.png)

But ideally, the code would in some way resemble the eventual Angular code, eg:

```html
<app-button class="button--light" linkUrl="https://www.google.com" (click)="doSomething()"></app-button>
```

I tried to approach this by first moving towards having it at least display something close to `class="button--light" linkUrl="https://www.google.com" (onclick)="doSomething()`, by generating a template literal function `ng`, which would take that string and convert it into the equivalent of `Button({ class: 'button--light', linkUrl: 'https://www.google.com' })`.

```ts
const Button = makeStory(ButtonsComponent);
const ng = makeTaggedTemplate(Button);

export const Light = ng`class="button--light" linkUrl="https://www.google.com"`;
```

When I do that, however, Storybook renders the component just fine, but fails to show my code:

![image](https://user-images.githubusercontent.com/1746778/80846278-89cb3880-8bc0-11ea-842a-efeb2c620db5.png)

Is there any way to get this to happen? (Eg, to have the "Show Code" for Angular in some way approximate what the actual component code would look like in a template?)

Thanks!

**Not sure system stuff is relevant, but:**

```
  System:
    OS: macOS Mojave 10.14.6
    CPU: (8) x64 Intel(R) Core(TM) i7-8559U CPU @ 2.70GHz
  Binaries:
    Node: 10.19.0 - ~/.nvm/versions/node/v10.19.0/bin/node
    Yarn: 1.22.4 - ~/.nvm/versions/node/v10.19.0/bin/yarn
    npm: 6.14.4 - ~/.nvm/versions/node/v10.19.0/bin/npm
  Browsers:
    Chrome: 81.0.4044.129
    Firefox: 75.0
    Safari: 13.1
  npmPackages:
    @storybook/addon-docs: ^6.0.0-alpha.45 => 6.0.0-beta.0 
    @storybook/addons: ^6.0.0-alpha.45 => 6.0.0-beta.0 
    @storybook/angular: ^6.0.0-alpha.45 => 6.0.0-beta.0 
```

**Comment by shilman** [comment]
Tracked in https://github.com/storybookjs/storybook/issues/11418

**Comment by shilman** [comment]
this is done and available in 6.2 thanks to @yngvebn

**Comment by samvienne** [comment]
Hi, I've read a lot of things but didn't found a clear solution about this problem. Is it possible to render the component temptate code into the source documentation ? Thanks for you time
